src/ethereum/frontier/state.py

```diff
@@ -421,3 +421,23 @@ def write_code(sender: Account) -> None:
         sender.code = code
 
     modify_state(state, address, write_code)
+
+
+def create_ether(state: State, address: Address, amount: U256) -> None:
+    """
+    Add newly created ether to an account.
```

Comment:
We'll have to change this docstring if we rename this to `add_to_account_balance`